src/import/FSHImporter.ts

```diff
@@ -130,9 +133,11 @@ export class FSHImporter extends FSHVisitor {
   private currentFile: string;
   private currentDoc: FSHDocument;
   private allAliases: Map<string, string>;
+  paramRuleSets: Map<string, ParamRuleSet>;
```

Comment:
Right now, I only have it visible so that I can check it in the tests. But, you're right that it makes sense for it to be private. Do you think it would be better to add a public method for test support rather than having the field itself be public?

Reply:
I'm not too worried about it, since I don't think we will start using `FSHImporter` other places outside of testing. So I'm fine with it as is, just wanted to understand why.